extensions/blocks/calendly/attributes.js

```diff
@@ -11,6 +11,9 @@ const colourValidator = value => hexRegex.test( value );
 const urlValidator = url => ! url || url.startsWith( 'https://calendly.com/' );
 
 export default {
+	align: {
```

Comment:
I don't know if we need this, since we don't set a default value.

Reply:
We do, without it, the alignment attribute doesn't work.

Reply:
Actually I checked this again, and you're right we don't need it. I've removed it.